user-guide/aragon-court/aragon-court-glossary.html

```diff
@@ -0,0 +1,124 @@
+<p>
+	<strong></strong><em>Below is a list of unique terms that you may encounter when using Aragon Court. You can learn more about these concepts and why they matter to you as an Aragon Court user by reading through the rest of the <a href="https://help.aragon.org/category/47-aragoncourt" target="_blank">Aragon Court juror documentation</a>.</em><strong><br>
+	</strong>
+</p>
+<p>
+	<strong>Appeal</strong>
+</p>
+<p>
+	When someone disagrees with the preliminary ruling of a dispute, they can challenge the outcome and escalate the dispute to the next round of jury voting. Each appeal round brings in a larger and larger group of jurors until the fourth and final appeal round, when all active jurors are drafted to vote on the final ruling.
```

Comment:
Not sure fourth is the final one; I think the current settings are four regular rounds with the fifth being the final one cc @facuspagnuolo.

Also not sure "all active jurors are drafted to vote on the final ruling" is correct. Technically, I count them as "invited"; if they don't vote in this round, they don't get penalized (unlike "drafted", where voting is a strict responsibility).

Reply:
> Not sure fourth is the final one; I think the current settings are four regular rounds with the fifth being the final one cc @facuspagnuolo.

That's an off-by-one error, you are correct :smile: good catch!

Reply:
> Also not sure "all active jurors are drafted to vote on the final ruling" is correct. Technically, I count them as "invited"; if they don't vote in this round, they don't get penalized (unlike "drafted", where voting is a strict responsibility).

Oh interesting, I thought it was a proper drafting! Can you confirm @facuspagnuolo?

Reply:
> That's correct but it says "fourth and final appeal round". You can appeal four times. There are five rounds in total (the first round plus four appeal rounds).

That's correct

> Technically, I count them as "invited"; if they don't vote in this round, they don't get penalized (unlike "drafted", where voting is a strict responsibility).

And this is correct as well